src/icons/_icons.scss

```diff
@@ -5,44 +5,46 @@
     overflow: visible;
     // The following is required to compensate for alignment
     // issues with the Bootstrap Icons alpha implementation
-    // And might be able to remove in future version of bootstrap-icons
-    // Perhaps this values should be SASS variables?
-    vertical-align: -0.15em;
+    // TODO: Is this "hack" still required?
```

Comment:
Yup still required for now...

Reply:
Yeah. The TODO ist for later to remind us.